_policy/referees.md

```diff
@@ -1,15 +1,448 @@
+required to ensure that the refereeing arm remains healthy, effective and in a positive state of
+regular development.
+
+### Aim
+
+The aim of this document is to detail the policy by which the refereeing arm of Touch is managed
+within and under the guidelines of FIT.
+
+### Authority
+
+This Referee Policy has been developed by the FIT Referee Commission and is published with the
+approval and endorsement of the FIT Board.
+
+### Review
+
+This FIT Referees Policy is a dynamic document. It requires review to ensure that it remains both
+current and applicable. The FIT Referees Commission is to review the policy annually. All
+recommended amendments are to be submitted to the Federation Board for endorsement.
+
+## Appointment of the FIT Referee Director of Referees & FIT Referee Commission
+
+### Appointment Protocol
+
+The following details apply to the appointment of the FIT Referee Director and to the
+appointments of FIT Referee Commission Members:
+
+1.  The term of appointment shall be from the date of approved by the FIT Board to one month after
+    the next election of Board members at the World Cup (approximately four year cycle).
+    Appointments made during the four year cycle and between Annual General Meetings will conclude
+    as per the timeframe detailed above. In the event that the FIT Board does not appoint the FIT
+    Referee Director and FIT Referee Commission Members prior to the expiry of the terms of
+    appointment, the FIT Referee Director and FIT Referee Commission Members will continue in office
+    past any expiry date of appointment to maintain continuity of service until the FIT Board makes
+    new appointments.
+2.  The respective appointment period commences post World Cup e.g. from World Cup 2015 in Australia
+    (May 2015) through to the 2019 World Cup.
+3.  Nominations should be endorsed by the Nominee's Member NTA (President, Chairperson, or CEO) OR
+    maybe endorsed by regional members of the Nominee's NTA, such as Regional President, Chairperson
+    or CEO.
+4.  Nominations must be submitted to the FIT Secretary General via email to :
+    secretary‑general@internationaltouch.org
+5.  Successful applicants will receive letters of appointment from the Secretary-General of FIT
+    following initial telephone or e-mail confirmation.
+6.  The FIT Board will determine the process of appointment for the FIT Director of Referees and
+    Commission Members taking into account the requirement for a cost efficient and effective
+    process of appointment to be used.  This could include (but should not be limited to) an
+    appointment being made on the basis of application only without interview or an interview via a
+    telecommunications method.
+7.  Application for the above position will be advertised on the FIT website. All policy documents
+    will be available on the FIT website.
+
+### FIT Director of Referees Selection Criteria
```

Comment:
Is this required in the policy? It is more or less restating the position description of the [Referees Commission Chair](http://docs.internationaltouch.org/position/referees-commission/chair), but not as well.